kahuna/public/js/components/gr-panel/gr-panel.html

```diff
@@ -13,348 +13,348 @@
         </button>
     </div>
 
-    <div ng:if="ctrl.selectedImages.size > 0">
```

Comment:
I decided to remove the extra `div` that caused issues, and move the `ng:if` onto both children. We could have kept the `div` but then would have to apply `height: 100%` onto it, which makes it an HTML+CSS fix that may be more likely to be broken again in the future (fix implies tight coupling of code far away). The HTML-only fixes seemed less brittle.